docs/static/security/logstash.asciidoc

```diff
@@ -18,14 +18,10 @@ pipeline is halted if authentication fails.
 If encryption is enabled on the cluster, you also need to enable TLS/SSL in the
 Logstash configuration.
```

Comment:
This is no longer an "if encryption is enabled" - Should re-phrase . Probably a continuation of the previous phrase. "Because security is on by default, you will also need to enable TLS/SSL in the Logstash configuration in order for Logstash to communicate to the cluster"